types/hand-crafted/JitsiMeetJS.d.ts

```diff
@@ -59,33 +69,33 @@ export type JitsiMeetJSType = {
   //USER_MEDIA_SLOW_PROMISE_TIMEOUT: 1000;
 
   constants: {
-    participantConnectionStatus: ParticipantConnectionStatus,
+    participantConnectionStatus: typeof ParticipantConnectionStatus,
     recording: typeof RecordingConstants,
-    sipVideoGW: VideoSIPGWConstants,
-    transcriptionStatus: JitsiTranscriptionStatus,
+    sipVideoGW: typeof VideoSIPGWConstants,
+    transcriptionStatus: typeof JitsiTranscriptionStatus,
   };
 
   events: {
-    conference: JitsiConferenceEvents,
-    connection: JitsiConnectionEvents,
-    detection: DetectionEvents,
-    track: JitsiTrackEvents,
-    mediaDevices: JitsiMediaDevicesEvents,
-    connectionQuality: ConnectionQualityEvents,
-    e2eping: E2ePingEvents
+    conference: typeof JitsiConferenceEvents,
```

Comment:
Why do we need `typeof` here?

Reply:
We want to access the members of the enum through `conference` like `JitsiMeetJS.events.conference.someEnumMember` therefore we need the typeof operator here.